pom.xml

```diff
@@ -62,22 +62,22 @@
 
         <!-- versions -->
         <zookeeper-version>3.4.8</zookeeper-version>
-        <maven-project-info-reports-plugin-version>2.7</maven-project-info-reports-plugin-version>
-        <maven-bundle-plugin-version>2.3.7</maven-bundle-plugin-version>
-        <maven-javadoc-plugin-version>2.10.3</maven-javadoc-plugin-version>
-        <doxia-module-confluence-version>1.6</doxia-module-confluence-version>
+        <maven-project-info-reports-plugin-version>2.9</maven-project-info-reports-plugin-version>
+        <maven-bundle-plugin-version>3.2.0</maven-bundle-plugin-version>
+        <maven-javadoc-plugin-version>2.10.4</maven-javadoc-plugin-version>
+        <doxia-module-confluence-version>1.7</doxia-module-confluence-version>
         <maven-license-plugin-version>1.9.0</maven-license-plugin-version>
-        <javassist-version>3.18.1-GA</javassist-version>
+        <javassist-version>3.20.0-GA</javassist-version>
         <commons-math-version>2.2</commons-math-version>
         <jackson-mapper-asl-version>1.9.13</jackson-mapper-asl-version>
         <jersey-version>1.18.1</jersey-version>
         <jsr311-api-version>1.1.1</jsr311-api-version>
         <jetty-version>6.1.26</jetty-version>
         <scannotation-version>1.0.2</scannotation-version>
         <resteasy-jaxrs-version>2.3.0.GA</resteasy-jaxrs-version>
-        <guava-version>16.0.1</guava-version>
+        <guava-version>19.0</guava-version>
```

Comment:
As we now shade Guava into Curator I don't think it's that big of a deal which version we put in our POM. I'd say use the most commonly used version so that we don't screw up anyone else's build.